src/main/java/org/zalando/nakadi/domain/SubscriptionEventTypeStats.java

```diff
@@ -27,38 +26,13 @@ public String getEventType() {
     }
 
     @Immutable
+    @Getter
+    @AllArgsConstructor
     public static class Partition {
         private final String partition;
         private final String state;
         @JsonInclude(JsonInclude.Include.NON_NULL)
         private final Long unconsumedEvents;
         private final String streamId;
-
-        public Partition(final String partition,
-                         final String state,
-                         @Nullable final Long unconsumedEvents,
-                         final String streamId) {
-            this.partition = partition;
-            this.state = state;
-            this.unconsumedEvents = unconsumedEvents;
-            this.streamId = streamId;
-        }
-
-        public String getPartition() {
-            return partition;
-        }
-
-        public String getState() {
-            return state;
-        }
-
-        @Nullable
```

Comment:
@rcillo if this rule is also strictly followed for Getters and Setters, then i do not think this can be done using lombok. The framework just assumes those methods are to be generated in a conventional way. Think i will close this PR, thanks for mentioning this rule.